Make reachable/unreachable condition more readable

vdemeester opened the pull request:
Introducing methods to make the intent of the condition clearer to the eyes of the reader 👼.

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

aaronlehmann commented on daemon/cluster/cluster.go:
```diff
@@ -509,6 +509,14 @@ func (c *Cluster) stopNode() error {
 	return nil
 }
 
+func atLeastOneMoreNodeReachable(reachable, unreachable int) bool {
```
I don't think this name accurately describes what it's doing. I believe it's checking for a situation where removing the manager would cause a loss of quorum.

vdemeester replied:
hum true, I should change it to a more *domain* name :angel: 
Updating it :wink:

vdemeester committed:
Make reachable/unreachable condition more readable

Introducing methods to make the intent of the condition clearer to the
eyes of the reader 👼.

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

thaJeztah commented:
LGTM

cpuguy83 commented:
LGTM
